frame/vesting/src/lib.rs

```diff
@@ -127,6 +141,20 @@ pub mod module {
 
 		/// The time provider.
 		type Time: Time<Moment = Self::Moment>;
+
+		/// The ID of a vesting schedule.
+		type VestingScheduleId: Copy
+			+ Clone
+			+ Eq
+			+ Debug
+			+ Zero
+			+ SafeAdd
+			+ One
+			+ Ord
+			+ FullCodec
+			+ MaxEncodedLen
+			+ MaybeSerializeDeserialize
+			+ TypeInfo;
```

Comment:
Consider using [this](https://docs.rs/frame-support/latest/frame_support/pallet_prelude/trait.Member.html) and then add any other bounds you need